native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -0,0 +1,297 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  Image,
+  Platform,
+  StyleSheet,
+  TouchableHighlight,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+import BpkText from 'react-native-bpk-component-text';
+
+const tickIcon = require('./icons/lg/tick-circle.png'); // eslint-disable-line import/no-unresolved
+const informationIcon = require('./icons/lg/information-circle.png'); // eslint-disable-line import/no-unresolved
+const closeIcon = require('./icons/lg/close.png'); // eslint-disable-line import/no-unresolved
+const chevronUpIcon = require('./icons/lg/chevron-up.png'); // eslint-disable-line import/no-unresolved
+const chevronDownIcon = require('./icons/lg/chevron-down.png'); // eslint-disable-line import/no-unresolved
```

Comment:
How about we use `BpkIcon` here 😄 ?

Reply:
⬆️ https://github.com/Skyscanner/backpack/pull/156#discussion_r145050555